packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -271,3 +297,14 @@ async function sendTrace(
     logger.error('Failed to send trace:', err);
   }
 }
+
+function isDocumentNode(data: unknown): data is DocumentNode {
+  const isObject = (data: unknown): data is Record<string, unknown> =>
```

Comment:
Not sure this has to be inlined into the `isDocument` function ? It's not using any variable from the scope.

Reply:
It is not used anywhere else and is only for internal purposes of isDocument method. But if you want, I will define it in the root of this file.